packages/frontend/src/presenter/DataSources/constants.ts

```diff
@@ -83,3 +84,11 @@ export const TRANSFORMS_LABELS: { [key: string]: string } = {
   namingConvention: 'Naming Convention',
   resolversComposition: 'Resolvers Composition',
 };
+
+export const CUSTOM_HANDLER_TO_PACKAGE: Record<string, string> = {
+  ContentfulHandler: '@graphql-portal/contentful',
```

Comment:
Looks scary :) Why do we need that?

Reply:
We will need this mapping tables for the next PR :)
GraphQL-mesh searches all custom handlers by the package name. Thats why we need to send them as:
```
handler: {
  'PACKAGE_NAME': { ...HANDLER_BODY }
}

handler: {
  '@graphql-portal/contentful': {
    'token': CONTENTFUL_TOKEN,
    'endpoint': CONTENTFUL_ENDPOINT
  }
}
```